bin/mustache_cli.ml

```diff
@@ -12,18 +12,23 @@ let load_file f =
   close_in ic;
   (Bytes.to_string s)
 
-let run json_filename template_filename =
+let run ?(prepend_json=ref false) json_filename template_filename =
   let j = load_file json_filename
-  and t = load_file template_filename
-  in
-  print_endline j;
-  apply_mustache j t
-
-let usage () =
-  print_endline "Usage: mustache-cli json_filename template_filename"
+  and t = load_file template_filename in
+  let result = apply_mustache j t in
+  (match prepend_json.contents with
+   | true -> [j; result] |> String.concat ""
+   | false -> result)
+  |> print_endline
```

Comment:
This is too complex. What about `if prepend_json then print_endline j;` followed by printing the result?